src/components/plots/helpers/heatmap/generateVitessceHeatmapTracksData.js

```diff
@@ -0,0 +1,27 @@
+import getCellClassProperties from 'utils/cellSets/getCellClassProperties';
+import { hexToRgb } from 'components/plots/helpers/heatmap/utils';
+
+const generateVitessceHeatmapTracksData = (trackOrder, hierarchy, properties, cells) => {
```

Comment:
this function is only used in `src/components/plots/helpers/heatmap/populateHeatmapData.js`, so why don't you move it to be there directly?

Reply:
I feel like `populateHeatmapData` already is way too large, ideally I would even think about moving those away into utils or separate files to make it easier to follow

Reply:
I did some ordering and separation into files/structure, we can go over it and you give me your opinion